web/src/filter.js

```diff
@@ -1095,4 +1095,16 @@ export class Filter {
         }
         return true;
     }
+
+    is_conversation_view() {
+        const term_type = this.sorted_term_types();
+        if (
+            _.isEqual(term_type, ["stream", "topic"]) ||
+            _.isEqual(term_type, ["pm-with"]) ||
+            _.isEqual(term_type, ["group-pm-with"])
```

Comment:
group-pm-with is not a conversation view; you drop it from this list.

Reply:
Ahh okay that's interesting, I thought `group-pm-with` meant group chats but it's really just list of messages in private chat where that person is involved in.